lib/vsc/install/shared_setup.py

```diff
@@ -24,19 +24,46 @@
 import shutil
 import sys
 import re
+
+try:
+    from unittest.suite import TestSuite
+except ImportError:
+    from unittest import TestSuite
```

Comment:
the fallback is for older Python versions or something? add a comment to clarify please